application-mocca-calendar-ui/src/main/resources/MoccaCalendar/MoccaCalendarEventSheet.xml

```diff
@@ -291,6 +340,14 @@ $doc.display($field)
       &lt;dd&gt;
         $request.description
       &lt;/dd&gt;
+      &lt;dt&gt;&lt;label for="backgroundColor"&gt;$escapetool.xml($services.localization.render('MoccaCalendar.MoccaCalendarEventClass_backgroundColor'))&lt;/label&gt;&lt;/dt&gt;
```

Comment:
This line is too long (>120 characters)
The idea of keeping lines in the 120ch limit is to have it in your computer visual limit, so you don't have to scroll horizontally when looking at the code
Below as well